Modules/socketmodule.c

```diff
@@ -8187,6 +8187,48 @@ socket_exec(PyObject *m)
 
     ADD_INT_MACRO(m, J1939_FILTER_MAX);
 #endif
+#ifdef HAVE_LINUX_CAN_ISOTP_H
+    ADD_INT_MACRO(m, SOL_CAN_ISOTP);
+
+    /* for socket options affecting the socket (not the global system) */
```

Comment:
Consider something more concise.

```suggestion
    /* Options affecting the socket (not the global system). */
```